ROADMAP.md

```diff
@@ -1,24 +1,72 @@
 # PlasmaPy Development Roadmap
 
-This document summarizes the overall development plans for PlasmaPy.
-This roadmap is very fluid and is subject to change due to community
-priorities.  For the most up-to-date information, see PlasmaPy's
-[issues](https://github.com/PlasmaPy/plasmapy/issues), [pull
-requests](https://github.com/PlasmaPy/plasmapy/pulls), and
-[projects](https://github.com/PlasmaPy/PlasmaPy/projects).
+This document summarizes the long-term development plans for PlasmaPy.
+This roadmap is very fluid and will continue to be updated and changed
```

Comment:
So it's a rather collisional roadmap, then?

Reply:
I was very tempted to say `This roadmap is very plasma` instead but decided against it.